### PR TITLE
Add network status indicator

### PR DESCRIPTION
**Pull request by nazar-pc**
This is an MVP version of https://github.com/autonomys/space-acres/issues/340, the whole thing is a link to documentation about P2P ports

Bad
Okay

### CODE REVIEW

**Comment by nazar-pc** [comment]
@randywessels is this sufficient to close https://github.com/autonomys/space-acres/issues/340 for now?

**Comment by randywessels** [comment]
> @randywessels is this sufficient to close #340 for now?

I think it is a great starting point. Perhaps we can add on to it later, but waiting for feedback and seeing how much it helps in troubleshooting makes a lot of sense.

File: res/translations/fr/messages.ftl [hunk 1]
```diff
@@ -96,6 +96,8 @@ configuration_dialog_button_cancel = Annuler
 running_title = En cours
 running_node_title = {$chain_name} Nœud de la blockchain
 running_node_title_tooltip = Cliquez pour ouvrir dans le gestionnaire de fichiers
+# TODO: Translate
+running_node_connections_tooltip = {$connected_peers}/{$expected_peers} peers connected, click for details about required P2P ports
```
Comment (Vlady19) [inline res/translations/fr/messages.ftl:100]: Pairs connectées, cliquez pour plus de détails sur les ports P2P requis

**Comment by Vlady19** [comment]
Pairs connectées, cliquez pour plus de détails sur les ports P2P requis

**Pull request merged**